red2.dSYM/Contents/Info.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm assuming you didn't mean to check in this file

Reply:
Oops, thanks